src/components/dataDisplays/IndividualsDisplay.jsx

```diff
@@ -31,6 +33,15 @@ export default function IndividualsDisplay({
         ),
       },
     },
+    {
+      name: 'role',
+      labelId: 'ROLE',
```

Comment:
I dont _think_ you would put relationship role in tables. Because that only makes sense in the context of the two individuals in the relationship. Maybe I'm wrong, but I think we can disambiguate if/when we add relationship roles